lib/lita/handlers/jira.rb

```diff
@@ -90,6 +99,14 @@ def comment(response)
         response.reply(t('comment.added', issue: issue.key))
       end
 
+      def point(response)
+        issue = fetch_issue(response.match_data['issue'])
+        return response.reply(t('error.request')) unless issue
+        points = response.match_data['points']
+        issue.save(fields: { customfield_10004: points.to_i }) # Story points is a custom field with id 10004
```

Comment:
I just realized this is unique to our JIRA setup. Any thoughts on how to parameterize this?